tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py

```diff
@@ -144,97 +179,131 @@ def language_menu_item_description(self):
-        return FindMyDevice(self.marionette)
-
-    def open_device_info_settings(self):
-        from gaiatest.apps.settings.regions.device_info import DeviceInfo
-        self._tap_menu_item(self._device_info_menu_item_locator)
-        return DeviceInfo(self.marionette)
-
-    def open_battery_settings(self):
-        from gaiatest.apps.settings.regions.battery import Battery
-        self._tap_menu_item(self._battery_menu_item_locator)
-        return Battery(self.marionette)
-
-    def open_sim_manager_settings(self):
-        from gaiatest.apps.settings.regions.sim_manager import SimManager
-        self._tap_menu_item(self._sim_manager_menu_item_locator)
-        return SimManager(self.marionette)
-
-    def open_homescreen_settings(self):
-        from gaiatest.apps.settings.regions.homescreen_settings import HomescreenSettings
-        self._tap_menu_item(self._homescreen_menu_item_locator)
-        return HomescreenSettings(self.marionette)
-
-    def open_browsing_privacy_settings(self):
-        from gaiatest.apps.settings.regions.browsing_privacy import BrowsingPrivacy
-        self._tap_menu_item(self._browsing_privacy_item_locator)
-        return BrowsingPrivacy(self.marionette)
+        return self._open_subpage(self._findmydevice_locator, 'findmydevice', 'FindMyDevice')
+
+    def open_screen_lock(self):
+        return self._open_subpage(self._screen_lock_menu_item_locator, 'screen_lock', 'ScreenLock')
+
+    def open_app_permissions(self):
+        return self._open_subpage(self._app_permission_menu_item_locator)
+
+    def open_do_not_track(self):
+        return self._open_subpage(self._do_not_track_menu_item_locator, 'do_not_track', 'DoNotTrack')
+
+    def open_browsing_privacy(self):
+        return self._open_subpage(self._browsing_privacy_item_locator, 'browsing_privacy', 'BrowsingPrivacy')
+
+    def open_media_storage(self):
+        return self._open_subpage(self._media_storage_menu_item_locator, 'media_storage', 'MediaStorage')
+
+    def open_application_storage(self):
+        return self._open_subpage(self._application_storage_menu_item_locator)
+
+    def open_device_info(self):
+        return self._open_subpage(self._device_info_menu_item_locator, 'device_info', 'DeviceInfo')
+
+    def open_downloads(self):
+        return self._open_subpage(self._downloads_menu_item_locator)
+
+    def open_battery(self):
+        return self._open_subpage(self._battery_menu_item_locator, 'battery', 'Battery')
+
+    def open_accessibility(self):
+        return self._open_subpage(self._accessibility_menu_item_locator, 'accessibility', 'Accessibility')
+
+    def open_developer(self):
+        return self._open_subpage(self._developer_menu_item_locator)
+
+    def open_improve(self):
+        return self._open_subpage(self._improve_menu_item_locator)
+
+    def open_help(self):
+        return self._open_subpage(self._help_menu_item_locator)
+
+    def _open_subpage(self, locator, file_name=None, class_name=None):
```

Comment:
I thought about parsing the file name to get the class name, but since this implicitly assumes that we would always follow this convention, which might not be true.  I didn't want to make an assumption when it involves filesystem names, just to be safe.